docs/cloud.md

```diff
@@ -0,0 +1,59 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in some cloud managed clusters. This document describes
+ steps to create EKS using terraform.
+
+## Common Pre-requisites
+1. To run EKS cluster, install and configure aws cli, see
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html, and
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+2. Install aws-iam-authenticator, https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam
+-authenticator.html
+3. Install terraform
+   see https://learn.hashicorp.com/terraform/getting-started/install.html
+4. You must already have ssh key-pair created. This key pair will be used to access worker node via
+ ssh.
+```bash
+ls ~/.ssh/
+id_rsa  id_rsa.pub
+```
+
+
+## Create an EKS cluster via terraform
+Ensures that you have permission to create EKS cluster, and have already
+created EKS cluster role as well as worker node profile.
+```bash
+export TF_VAR_eks_cluster_iam_role_name=YOUR_EKS_ROLE
+export TF_VAR_eks_iam_instance_profile_name=YOUR_EKS_WORKER_NODE_PROFILE
+export TF_VAR_eks_key_pair_name=YOUR_KEY_PAIR_TO_ACCESS_WORKER_NODE
+```
+Where 
+- TF_VAR_eks_cluster_iam_role_name may be created [here](https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html#create-service-role)
```

Comment:
s/may be created [here]/may be created by following [these instructions]

is more accurate, same below

Reply:
done